1/Dockerfile

```diff
@@ -48,6 +48,36 @@ RUN /usr/local/bin/plugins.sh /opt/openshift/base-plugins.txt && \
     touch /opt/openshift/plugins/subversion.jpi.pinned && \
     touch /opt/openshift/plugins/ssh-credentials.jpi.pinned && \
     touch /opt/openshift/plugins/script-security.jpi.pinned && \
+    chmod 775 /etc/passwd && \
```

Comment:
@bparees and I had a confab with Clayton ... that coupled with some experimentation leads us to believe we are OK, but are still going to sit and ponder for a bit since this PR is post 3.5.